Comm_Demo.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
when we get so many example notebooks, maybe it would be best to move all 4 into a common directory.

be sure to add the directory to .RBuildignore